### DISCUSSION TITLE
[Question]: 0.14 vs 0.13 detected bounding boxes in recordings?

### DISCUSSION THREAD

**Discussion by majorsl** [blakeblackshear/frigate#13115] [Ask A Question] [answered]
### What is your question:

In 0.13, when reviewing clips, you could hit a button and see a bounding box for what it thought it detected. The was amazingly helpful in figuring out weird and sometimes random false positives to then exclude areas. Is this available in 0.14 and I'm just not seeing it or is there a config for it?

**Comment by NickM-27** [comment, marked as answer]
See https://github.com/blakeblackshear/frigate/issues/11399

**Reply by majorsl** [reply]
Oh excellent. Hopefully we'll get a quick ..1 release to take care of a few of these initial issues!
